### ISSUE TITLE
Unable to focus on modal dialogs.

### ISSUE THREAD

**Issue by blindndangerous** [AzuraCast/AzuraCast#5519]
### Installation Method

Docker Installation

### AzuraCast Release Channel

Rolling Release Channel

### Current AzuraCast Version

Powered by AzuraCast • Rolling Release #5b46d13 (2022-06-20 0:27) • Docker • PHP 8.1

### What happened?

When clicking on the link that says reload to apply changes in a stations profile, a dialog comes up saying that this will not disconnect any listeners and to either reload or cancel.  However, I cannot access this dialog with my screenreader, as pressing enter on it does not drop me into it.  Pressing escape also does not close the modal, the only way to close it is to click the cancel button.

**Comment by ItsMitchh** [comment]
Hello, I will be looking into this now.

**Comment by ItsMitchh** [comment]
Hi there @blindndangerous can you provide the name of the screenreader you are using? This will help us look into this issue further.

**Comment by blindndangerous** [comment]
I use nvda, https://www.nvaccess.org/

**Comment by ChrisDuffley** [comment]
@blindndangerous, thanks for making me aware of this as I'm also an NVDA and JAWS user, so I'll be waiting for another commit that fixes this issue before updating. Also, to make you aware, certain browsers like Brave can also have this issue but Chrome may not, for instance, so knowing what you're using for a browser would also be helpful.

**Comment by blindndangerous** [comment]
That is true.  I'm using Chrome.  I didn't try with Firefox.

**Comment by SlvrEagle23** [comment]
I admit I'm having trouble reproducing this issue.

When the modal pops up, by default the browser focuses on one of the buttons (either "OK" or "Cancel"), depending on whether the action is dangerous or not. This autofocus behavior may be what's throwing off the screen reader's ability to read the entire thing...it may be more helpful for us to not focus any button, but we consider it a favorable UI for users looking to do things quickly that we focus on the button by default.
